README.md

```diff
@@ -0,0 +1,83 @@
+# PyVista Tutorial
+
+## Abstract
+
+This tutorial will introduce 3D data visualization using PyVista (https://github.com/pyvista/pyvista).
+PyVista offers 3D plotting and mesh analysis through a streamlined interface for the Visualization Toolkit (VTK).
+PyVista uses VTK (http://www.vtk.org) under the covers and provides a very Python-friendly API.
+
+At the end of the tutorial users will be able to
+
+- Use PyVista effectively to create 3D visualizations. 
+- Understand the basic class structure of PyVista.
+
+## Keywords
+
+[visualization](https://github.com/topics/visualization)
+[meshviewer](https://github.com/topics/meshviewer)
+[vtk](https://github.com/topics/vtk)
+[3d](https://github.com/topics/3d)
+
+## Other Information and Files
+
+### Tutorial Description.
+Same as Abstract.
+
+### Tutorial Outline.
+
+0. Getting Started - Why PyVista? (10 min for talk, 5 min for train)
+
+1. Basic usage - Reading and plotting 3D data.  (10 min for talk, 5 min for train)
+
+2. What is a Mesh? - Learn about "Point", "Cell" and "Attributes". (10 min for talk, 5 min for train)
+
+3. PyVista Data Model - Learn about the each data model of the PyVista framework. (10 min for talk, 5 min for train)
+
+4. Mesh Creation - Train to build and plot meshes in PyVista. (10 min for talk, 5 min for train)
+
+5. Camera/Lights API - Train to use camera/light objects to customize your plot. (10 min for talk, 5 min for train)
```

Comment:
I wonder if we (you) should include these in the tutorial. On one hand they are pretty advanced topics, far from the hands-on "just plot some stuff real quick" approach. On the other hand especially lighting can give some really compelling examples of what PyVista can do.

Reply:
The content is certainly too advanced as you say. Let's delete this section to make more time for the other sections.